Content/build.fsx

```diff
@@ -63,23 +63,16 @@ Target.create "Azure" (fun _ ->
     |> ignore
 )
 
-Target.create "Run" (fun _ ->
+let run clientCommand sharedPath serverPath _ =
```

Comment:
not convinced by this change - yes it makes script shorter, but adds another layer of complexity

Reply:
I must admit I'm not 100% sure of it either. I did it because I wanted to add the --no-restore for both. But now I've realised that you still need it because the build command doesn't run against the Server project, it runs against Shared. Which means that when you do a watch run against the server, it fails.

So maybe just bin this one.